astropy/wcs/wcs.py

```diff
@@ -1031,6 +1032,16 @@ def _read_sip_kw(self, header):
 
             del header[str('A_ORDER')]
             del header[str('B_ORDER')]
+            ctype=[header['CTYPE{0}'.format(nax)] for nax in range(1, self.naxis + 1)]
+            if any([ctyp[-4 :] != '-SIP' for ctyp in ctype]):
```

Comment:
It seems that SIP Paper (2008) specifies that '-SIP' is appended to CTYPE1 and CTYPE2. It is not clear if '-SIP' suffix is appended to CTYPE3 if it is a spectral axis.